src/styles/base.scss

```diff
@@ -4,11 +4,32 @@
 @import './mixins.scss';
 @import './grid.scss';
 @import './font.scss';
+@import url('https://fonts.googleapis.com/css?family=Roboto:300,400,500,700');
```

Comment:
So reason for not using CDN for font is it's display property `SWAP`. Basically to render default font until main font loaded we use `font-display:swap` which is not present in CDN. That's why we were using downloaded font.

Reply:
That was just for your information. Let's make it CDN for now. Will check the performance later. @live-wire

Reply:
Interesting :octocat:  Didn't know about that 😄